chore(thermocycler-gen2): set up channel testing script

fsinapi opened the pull request:
This adds a self contained python testing script intended to test the performance of each individual channel on a Thermocycler.

Other misc changes:
- Added a command in `test_utils` to get the power for each peltier
- Added power data in a separate plot to the `plot_data` tool

Tested by performing a run on an EVT unit

fsinapi committed:
Added a "channel test" script to get temperature data for each temperature while driving to target temperatures

fsinapi committed:
Fixed an argparse help line

amitlissack commented on stm32-modules/thermocycler-gen2/scripts/channel_test.py:
```diff
@@ -0,0 +1,170 @@
+    _POLL_FREQ = 2.0 
+    '''Poll frequency in Hz'''
+
+    class Thermistors():
```
The `@dataclass` decorator is a nice tool for these kinds of data classes.

amitlissack commented on stm32-modules/thermocycler-gen2/scripts/channel_test.py:
```diff
@@ -0,0 +1,170 @@
+
+class Thermocycler():
+    def __init__(self, port: str = None, debug: bool = False):
+        '''
```
Double quotes for docstrings

amitlissack reviewed (commented):
Just some little pythonic comments.

fsinapi committed:
Added a couple print statements, and edits per Amit's review:
- Single quote --> Double quote docstrings
- Take advantage of `@dataclass` decorator

sfoster1 reviewed (approved):
Looks good to me!
